### PR TITLE
S02 us19 create profil page and modification profil page

### PR DESCRIPTION
**Pull request by Thomas-Lunardo**

### CODE REVIEW

**Review by victorzanotelli — APPROVED** [review]

File: src/View/User/edit-user-profil.html.twig [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+				<header>
+					<label for="video">Entrer votre nouveau lien youtube</label>
+					<input type="text" value="{{ user.video }}" name="video" id="video">
+				</header>
+				<p>Votre vidéo actuelle</p>
+				{% if user.video %}
+					{% set videoId = user.video|split('/')|last|split('?')|first %}
+					{% set embedLink = 'https://www.youtube.com/embed/' ~ videoId %}
+
+					<iframe width="560" height="315" src="{{ embedLink }}" frameborder="0" allowfullscreen></iframe>
```
Comment (benrcd) [inline src/View/User/edit-user-profil.html.twig]: Style goes to CSS file.

File: src/View/User/user-profil.html.twig [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		</article>
+		<article class="video">
+			<header>
+				<p>Vidéo</p>
+			</header>
+			{% if user.video %}
+				{% set videoId = user.video|split('/')|last|split('?')|first %}
+				{% set embedLink = 'https://www.youtube.com/embed/' ~ videoId %}
+
+				<iframe width="560" height="315" src="{{ embedLink }}" frameborder="0" allowfullscreen></iframe>
```
Comment (benrcd) [inline src/View/User/user-profil.html.twig]: Same here

**Review by benrcd — CHANGES_REQUESTED** [review]

File: src/View/User/edit-user-profil.html.twig (hunk 1 above)
Comment (Thomas-Lunardo) [reply src/View/User/edit-user-profil.html.twig]: Done

File: src/View/User/user-profil.html.twig (hunk 2 above)
Comment (Thomas-Lunardo) [reply src/View/User/user-profil.html.twig]: Done

**Review by victorzanotelli — APPROVED** [review]

**Review by Merhene — APPROVED** [review]

**Review by benrcd — APPROVED** [review]

**Pull request merged**